### DISCUSSION TITLE
İnternet sitelerine get işlemi

### DISCUSSION THREAD

**Discussion by BunyaminAkdeniz** [19/classroom#54] [Anonim] [answered]
Merhaba Hocam. Bir döviz takip uygulaması yapıyorum fakat siteden istediğim api 20 get işleminden sonra fiyatlandırılıyor. Bende geçici olarak direkt site üzerinden kurguladım. 
```ruby
require 'rest-client'
site = RestClient.get("https://finanswebde.com/doviz/USD")
puts site.body.match(/"full_name":"Amerikan Doları","latest":....../)
```
Bu tür kullanımlarda bir protokol ihlali söz konusu olur mu?

**Comment by roktas** [comment, marked as answer]
:-)  Bu tabii bir tür kaçak kullanım.  Kişisel bilgilenme için olabilir ama başkalarına sunacağın bir servisin arka yüzü için pek sağlıklı görünmüyor.
